lib/alchemy/permissions.rb

```diff
@@ -142,15 +142,26 @@ def alchemy_editor_rules
         can [
           :copy,
           :copy_language_tree,
-          :create,
-          :destroy,
           :flush,
           :order,
-          :publish,
           :sort,
           :switch_language
         ], Alchemy::Page
 
+        # Resources which may be locked via template permissions
+        #
+        #     # config/alchemy/page_layouts.yml
+        #     - name: contact
+        #       editable_by:
+        #         - freelancer
+        #         - admin
+        #
+        can([
+          :create,
+          :destroy,
+          :publish
+        ], Alchemy::Page, Alchemy::Page) { |p| p.editable_by?(@user) }
```

Comment:
I was unsure if we also wanted to prevent the other resource permissions. I can keep them as one array though if it seems more sensible to block ALL actions to users without a matching role.